### ISSUE TITLE
`raw.line` doesn't do anything with unknown `lang` attribute

### ISSUE THREAD

**Issue by Zheoni** [typst/typst#2569]
### Description

I'm not sure if this is intended behaviour or not. In the docs it says `raw.line` is:
> A highlighted line of raw text

But the inconsistency comes from not setting a `lang` attribute runs `raw.line`.

In this example:
```typ
#show raw.line: it => [#text(red)[#it.number] #it.body]

#raw("= Known lang", lang: "typst", block: true)
#raw("Unknown lang", lang: "myweirdlang", block: true)
#raw("No lang", block: true)
```

The first and last ones add the line number, but not the second one. With what the docs say, I understand why it doesn't work with an unknown lang, but then it shouldn't work without the `lang` attribute either.

So... is it intented behaviour? If it is I think the docs should be updated to clarify this. Also an option could be added to always run `raw.line` no matter if the `raw` block is highlighted or not.

### Operating system

Web app, Windows

### Typst version

- [X] I am using the latest version of Typst

**Comment by PgBiel** [comment]
Culprit:
https://github.com/typst/typst/blob/8fd546760c7c425398f0114997c8085a481d8d2a/crates/typst-library/src/text/raw.rs#L364-L366

RawLine elements aren't being created when the language is not recognized.

@Dherse was this intentional?

**Comment by Dherse** [comment]
My bad I’ll fix it today.

**Comment by Dherse** [comment]
It's a big, it should still have wrapped every line into a raw.line I'll open a pr later today

**Comment by Dherse** [comment]
Fixed in #2504
